fhir_2_ada-r4/mp/9.3.0/payload/2.0.0-beta.3/mp-MedicationAdministration.xsl

```diff
@@ -0,0 +1,279 @@
+
+	<xd:doc>
+		<xd:desc>Template to convert f:MedicationAdministration to ADA medicatietoediening</xd:desc>
+	</xd:doc>
+	<xsl:template match="f:MedicationAdministration" mode="mp-MedicationAdministration">
+		<medicatietoediening>
+			<!-- identificatie -->
+			<xsl:apply-templates select="f:identifier" mode="#current"/>
+			<!-- toedienings_product -->
+			<xsl:apply-templates select="f:medicationReference" mode="#current"/>
+			<!-- toedienings_datum_tijd -->
+			<xsl:apply-templates select="f:effectiveDateTime" mode="#current"/>
+			<!-- afgesproken_datum_tijd -->
+			<!-- MP-1408 LR: afgesproken_datum_tijd nolonger supported from MP 9.3 beta.3 onwards but kept in stylesheet due to backwards compatibility-->
+			<xsl:apply-templates select="f:extension[@url = $urlExtMedicationAdministration2AgreedDateTime]" mode="#current"/>
+			<!-- geannuleerd_indicator -->
+			<xsl:if test="f:status/@value = 'entered-in-error'">
+				<geannuleerd_indicator value="true"/>
+			</xsl:if>
+			<!-- toegediende_hoeveelheid -->
+			<xsl:apply-templates select="f:dosage/f:dose" mode="#current"/>
+			<!-- afgesproken_hoeveelheid -->
+			<!-- MP-1395 LR: afgesproken_hoeveelheid removed -->
+			<!-- volgens_afspraak_indicator -->
+			<!-- TODO: should be updated in FHIR profile -->
+			<xsl:apply-templates select="f:extension[@url = $urlExtAsAgreedIndicator]" mode="#current"/>
+			<!-- toedieningsweg -->
+			<xsl:apply-templates select="f:dosage/f:route" mode="nl-core-InstructionsForUse"/>
+			<!-- toedieningssnelheid -->
+			<xsl:apply-templates select="f:dosage/f:rateQuantity" mode="#current"/>
+			<!-- prik_plak_locatie -->
+			<xsl:apply-templates select="f:dosage/f:site/f:text" mode="#current"/>
+			<!-- relatie medicatieafspraak -->
+			<xsl:apply-templates select="f:request" mode="#current">
+				<xsl:with-param name="outputMaOrWds">ma</xsl:with-param>
+			</xsl:apply-templates>
+			<!-- relatie_toedieningsafspraak -->
+			<xsl:apply-templates select="f:supportingInformation" mode="#current"/>
+			<!-- relatie_wisselend_doseerschema -->
+			<xsl:apply-templates select="f:request" mode="#current">
+				<xsl:with-param name="outputMaOrWds">wds</xsl:with-param>
+			</xsl:apply-templates>
+			<!-- relatie_contact of relatie_zorgepisode -->
+			<xsl:apply-templates select="f:context[f:reference | f:identifier]" mode="contextContactEpisodeOfCare"/>
+			<!-- toediener -->
+			<xsl:apply-templates select="f:performer" mode="#current"/>
+			<!-- medicatie_toediening_reden_van_afwijken -->
+			<xsl:apply-templates select="f:extension[@url = $urlExtMedicationAdministration2ReasonForDeviation]" mode="#current"/>
+			<!-- medicatie_toediening_status  -->
+			<!-- MP-1392 LR: medicatie_toediening_status nolonger supported from MP 9.3 beta.3 onwards but due to iets cardinality in FHIR (1..1) the value 'unknown' will result in medicatie_toediening_status being absent in ada -->
```

Comment:
Replace 'iets' by 'its'.

Reply:
I wonder whether this comment is correct. A `.status` with a value other than _unknown_ is still allowed (we don't fix that value in the FHIR profile), so it is still possible for there to be a medicatie_toediening_status element in ada.

Reply:
I added the same comment to .status as I did for .afgesproken_datum_tijd